models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -97,13 +97,14 @@ sipnet2datetime <- function(sipnet_tval, base_year, base_month = 1,
 ##' @param revision model revision
 ##' @param overwrite Flag for overwriting nc files or not
 ##' @param conflict Flag for dealing with conflicted nc files, if T we then will merge those, if F we will jump to the next.
+##' conflict is set to TRUE to enable the assimilation of monthly data.
 ##' @param prefix prefix to read the output files
 ##' @param delete.raw Flag to remove sipnet.out files, FALSE = do not remove files TRUE = remove files
 ##'
 ##' @export
 ##' @author Shawn Serbin, Michael Dietze
 model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date, delete.raw = FALSE, revision, prefix = "sipnet.out",
-                                overwrite = FALSE, conflict = FALSE) {
+                                overwrite = FALSE, conflict = TRUE) {
```

Comment:
Does it make sense to change the default here rather than just changing the value being passed in when your running the sub-annual SDA specifically?